net.sf.eclipsecs.core/build.properties

```diff
@@ -10,7 +10,7 @@ bin.includes = .,\
                lib/dom4j-2.1.3.jar, \
                lib/reflections-0.9.10.jar, \
                lib/snakeyaml-1.26.jar, \
-               lib/metadata-gen-1.0-SNAPSHOT.jar
+               lib/checkstyle-8.36-SNAPSHOT.jar
```

Comment:
Please use released version

And please remove build from Travis

Reply:
Done